Python/Blood Alcohol Content/README.md

```diff
@@ -0,0 +1,22 @@
+# Blood Alcohol Content
+
+Want to know how much amount of alcohol you drank? Well, the Blood alcohol content program will help you find out the alcohol intake.
```

Comment:
It would be a great idea to elaborate on the concept of BAC here as well. Not everyone understands the origin of the formula. PLease add more details